src/components/ContactForm.js

```diff
@@ -121,12 +121,14 @@ export const ContactForm = () => {
                 </Label>
                 <Button type="reset" onClick={handleReset} disabled={loading}>Reset</Button>
                 <Button type="submit" disabled={loading}>
-                    {loading ? <img src={loadingIcon} alt="loading"/> : 'Submit'}
+                    {loading ? <img src={loadingIcon} alt="Submitting"/> : 'Submit'}
```

Comment:
Always add a space before an autoclose
```suggestion
                    {loading ? <img src={loadingIcon} alt="Submitting" /> : 'Submit'}
```